includes/admin/feedzy-rss-feeds-admin.php

```diff
@@ -2369,4 +2366,48 @@ private function enable_telemetry() {
 		add_filter( 'themeisle_sdk_enable_telemetry', '__return_true' );
 		wp_enqueue_script( $this->plugin_name . '_telemetry', FEEDZY_ABSURL . 'js/telemetry.js', array(), $this->version, true );
 	}
+
+	/**
+	 * Set the black friday data.
+	 *
+	 * @param array $config The configuration array.
+	 * @return array
+	 */
+	public function set_black_friday_data( $config ) {
+		$product_label = __( 'Feedzy', 'feedzy-rss-feeds' );
+		$discount      = '70%';
+
+		// translators: %1$s - discount, %2$s - product label.
+		$config['message'] = sprintf( __( 'Our biggest sale of the year: <strong>%1$s OFF</strong> on <strong>%2$s</strong>! Don\'t miss this limited-time offer.', 'feedzy-rss-feeds' ), $discount, $product_label );
+
+		$is_pro      = feedzy_is_pro();
+		$license_key = '';
+
+		if ( $is_pro ) {
+			$product_label = __( 'Feedzy Pro', 'feedzy-rss-feeds' );
+			$discount      = '30%';
+
+			// translators: %1$s - discount, %2$s - product label.
+			$config['message'] = sprintf( __( 'Get <strong>%1$s off</strong> when you upgrade your <strong>%2$s</strong> plan or renew early.', 'feedzy-rss-feeds' ), $discount, $product_label );
```

Comment:
@Soare-Robert-Daniel we need also to use placeholders for html tags, no need to send them for translation.